packages/core/strapi/package.json

```diff
@@ -123,6 +123,7 @@
     "lodash": "4.17.21",
     "mime-types": "2.1.35",
     "node-fetch": "2.6.9",
+    "node-global-proxy": "1.0.1",
```

Comment:
Can we do without this lib?
It looks like an old one, not maintain and not really used. I had rather trust a bigger project 🤔 
https://www.npmjs.com/package/node-global-proxy (<2000 downloads a week)
https://github.com/wwwzbwcom/node-global-proxy (34 stars)

Reply:
I understand your point, but I think we cannot do it globally as it would mean it would apply to any requests that comes from the nodejs process and we may have some customer doing custom request to some internal servers in their custom code.

I think we have few request in the backend that goes to external servers, we may need to apply a proxy for each request manually.

Reply:
In my personal experience, doing things globally always ends up with more issues than it solves, especially when we talk about requesting external stuff.
I would suggest that we create a request service that acts as an adapter + middleware to inject (or not) a proxy configuration.
This way, we let the developers control what request is using the proxy or not.
WDYT?
I will try an implementation.

Reply:
I mean it can't hurt, at this point I don't really care how we solve it but more so when. We have a large number of customers having this issue and it keeps growing.

Reply:
What I'm trying to say is I believe a fix like this justifies a hot fix release so ideally we need to get this out as quickly as we possibly can.

Reply:
Shouldn't be a problem if they whitelist our domain, no?

Reply:
The problem is that the various fetch libs are not actually using the OS level proxy meaning we need application layer proxying.